CONTRIBUTING.md

```diff
@@ -255,7 +261,7 @@ For most development work, the account will be `cucumber`, which is created when
 
 Conjur has `rspec` and `cucumber` tests.
 
-Note on perfomance testing: [ci/docker-compose.yml](ci/docker-compose.yml) and
+Note on performance testing: [ci/docker-compose.yml](ci/docker-compose.yml) and
```

Comment:
Should be:
Note on performance testing: `WEB_CONCURRENCY: 0` is a configuration that is useful for recording accurate coverage data that can be used in the ci/docker-compose.yml and conjur/ci/authn-k8s/dev/dev_conjur.template.yaml. This isn't a realistic configuration and should do not use this for benchmarking.

@eranha why are even saying anything here if it shouldnt be used?

Reply:
It's greek to me,  I've just fixe the typo.